samples/quarkus/pet-store/src/main/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+quarkus.native.container-build=true
+quarkus.native.container-runtime=docker
+quarkus.native.builder-image=quay.azk8s.cn/quarkus/ubi-quarkus-native-image:19.2.1
```

Comment:
The Quarkus repo says we should be [pulling from quay.io](https://github.com/quarkusio/quarkus-images) - what is the azk8s.cn? Also, are we sure that this image will always generate executables for the correct target platform/libc? In the micronaut sample we fix on the [Docker image for Lambda's AMI](https://github.com/awslabs/aws-serverless-java-container/blob/master/samples/micronaut/pet-store/Dockerfile#L1)

Reply:
Quay.io is not accessible from mainland China. azk8s.cn is a local mirror for quay.io. I changed it to quay.io. 

This image works great to build native exectuabels for lambda custom time.